rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -399,7 +401,11 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 {
 	static Timer fps_t;
 
-	++m_frames;
+	if (++m_frames == 1u && !m_force_hide)
```

Comment:
m_frames gets resetted every 0.5 seconds, so it's not useable as an indicator for the actual first flip

Reply:
Fixed.